src/components/ReadingList/ReadingList.jsx

```diff
@@ -1,28 +1,23 @@
-import React, { Component } from 'react'
+import React from 'react'
 import readingListStyle from './readingList.module.css'
 
-class ReadingList extends Component {
-  render() {
-    const { items } = this.props
-    const bullets = items.map((x, index) => {
-      return (
-        <li key={`reading-list-item-${index}`}>
-          <a href={x.url}>{x.description}</a>
-        </li>
-      )
-    })
+export default function ReadingList({ items }) {
+  const bullets = items.map((x, index) => {
     return (
-      <div className={readingListStyle.readingList}>
-        <h3>
-          <span role="img" aria-hidden>
-            &#x1F4D6;
-          </span>{' '}
-          Further Reading
-        </h3>
-        <ul>{bullets}</ul>
-      </div>
+      <li key={`reading-list-item-${index}`}>
+        <a href={x.url}>{x.description}</a>
+      </li>
     )
-  }
```

Comment:
While we're here we could make this even shorter with an implicit return :)

```
const bullets = items.map((x, index)) => (<li> ... </li>)
```

Reply:
Will add this in too!

Reply:
looks so much nicer!